Formula/ship.rb

```diff
@@ -0,0 +1,28 @@
+class Ship < Formula
+  desc "Reducing the Overhead of Maintaining 3rd-party Applications in Kubernetes"
+  homepage "https://ship.replicated.com"
```

Comment:
it redirects to https://www.replicated.com/ship

I suggest to use it directly

Reply:
Thanks, @igas -- homepage URL is updated